ocs_ci/helpers/sanity_helpers.py

```diff
@@ -138,6 +94,9 @@ def delete_resources(self):
             pvc_obj.delete()
         for pvc_obj in self.pvc_objs:
             pvc_obj.ocp.wait_for_delete(pvc_obj.name)
+        for obc_obj in self.obc_objs:
+            obc_obj.delete()
```

Comment:
I forgot that `.delete()` has a `verify` parameter set to `True` by default, meaning the assert can actually be done on the deletion call itself 😅
Not a must at all.